.travis.yml

```diff
@@ -30,6 +30,9 @@ jobs:
   - stage: test
     node_js: 'lts/erbium'
```

Comment:
Can you update this to change erbium to fermium? This is the next codename for the 14 lts version ( https://github.com/nodejs/Release/blob/master/CODENAMES.md ).

This allows you to remove the newly added `node_js: 14`